springer-nature-link.md

```diff
@@ -1212,17 +1199,44 @@ Pages rendered in the <strong>newer</strong> design (currently articles, Chapter
 </ul></td>
 <td>Partially Supports</td>
 <td>
-This product contains content that is rendered in two separate designs. We are in the process of transferring all content from the older design to the newer design.
-<br><br>
-Pages rendered in the <strong>older</strong> design (currently journals, and ancillary pages such as search, indexes, and account management) use default browser focus rings only for interactive components.
+This product contains content that is rendered in two separate designs. We are in the process of transferring all content from the old design to the new design.
 <br><br>
-Pages rendered in the <strong>newer</strong> design (currently articles, chapters, books, book series, reference works, reference work entries, and collections) use highly-visible focus styles, including yellow focus rings or yellow backgrounds.
+All pages in this product <strong>except</strong> A-Z index pages are in the new design and use highly-visible focus styles, such as blue focus rings.
```

Comment:
I might mention that the blue focus rings don't always pass contrast in the footer or against the blue background for the advanced search link on the homepage under the search box